libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -448,20 +448,27 @@ class ofBaseVideoGrabber: virtual public ofBaseVideo{
 class ofBaseVideoPlayer: virtual public ofBaseVideo{
 
 public:
+	/// \brief Destroys the ofBaseVideoPlayer.
 	virtual ~ofBaseVideoPlayer();
 
-	/// \brief Load a video at the file path specified by \p name.
-	/// \param name The file path of the video you would like to load.
-	/// \returns Returns true if the video was loaded successfully.
-	/// \sa loadAsync
+	/// \brief Load a video resource by name.
```

Comment:
It might be worth noting that name here means file path, for beginners who might not quite understand that. Or maybe rather "load a video resource by name from the data/ folder". Or maybe that is too explicit...